docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '2'
 services:
   ui:
     tty: true # Enables debugging capabilities when attached to this container.
-    image: bitnami/angular:2.0.0
+    image: bitnami/angular:1.1.3-r0
```

Comment:
This is my fault, it's not actually a downgrade but I tagged it with the image version (as we normally do with containers) rather than the angular version. We should probably bump the image version to 2.0.0 to avoid the confusion